alby-nostr-wallet-connect/docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
       APP_PORT: 8080
       PROXY_AUTH_ADD: "false"
   server:
-    image: ghcr.io/getalby/nostr-wallet-connect:0.4.1@sha256:d1473f18ce706b542eaa7637dbab3bafa1e445aac32daffe3b31cbf91d329548
+    image: ghcr.io/getalby/nostr-wallet-connect:0.4.2@sha256:62fb85ee2ce46ffcfae44ddcf29163a4ddc6e36ae862256fb9784fadeb4e85ca
```

Comment:
The digest here needs to be changed to the multi-arch digest, which is 
`sha256:8e75c28eba2155ab1ad605e8d9c8de7e601dcc55d497e8641d37ae4cb3a4e433`

Right now, the current digest will result in a failed install for arm devices like the raspberry pi. Since images are on ghcr the easiest way to find your multiarch digest for nwc images is by looking at the main digest listed under a certain tag.

For example https://github.com/getalby/nostr-wallet-connect/pkgs/container/nostr-wallet-connect/168405078?tag=0.4.2:
<img width="594" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/215f4a7d-8988-4f18-a667-5d65edc2dcd1">

You can also run `docker pull <image>:<tag>` and the multi-arch digest (if available) will be output for you.

Reply:
Sorry about that, and thanks for pointing it out. 
Fixed now.